### PR TITLE
Use proxy to retrieve keys from github

### PR DESCRIPTION
**Pull request by bloodearnest**
Also add a test we can actually ssh in

### CODE REVIEW

File: scripts/update-ssh.sh [hunk 1]
```diff
@@ -19,7 +19,10 @@ if test -f "$cache" -a -n "${TEST}"; then
     cat "$cache" >> "$tmp"
 else
     # add current gh keys into $tmp
-    curl -s "https://github.com/$user.keys" >> "$tmp"
+    # note: a quirk of the github ssh api here is that a non-existant user
```
Comment (madwort) [inline scripts/update-ssh.sh:22]: nit: existant is french

File: Makefile [hunk 2]
```diff
@@ -42,11 +45,10 @@ test: $(TESTS)
 
 # run specific test
 .PHONY: $(TESTS)
-$(TESTS): .test-image $(CACHE_DIR)/updated
+$(TESTS): .test-image $(CACHE_DIR)/updated keys/testuser
 	./run-in-docker.sh $@
 
 
-# create a backend and drop you into a shell on it
```
Comment (madwort) [inline Makefile:49]: is this intentional removal of comment, seems unrelated?

File: tests/testuser [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# test version of developers file
```
Comment (madwort) [inline tests/testuser:1]: feel like this file could have a better name - `testdevelopers`? `developers-test`? etc?

Comment (bloodearnest) [reply tests/testuser:1]: tests/developers?

File: Makefile (hunk 2 above)
Comment (bloodearnest) [reply Makefile:49]: yeah, it was dangling bad comment

**Review by evansd — APPROVED** [review]
